apps/fabric-tester/ios/fabrictester.xcodeproj/project.pbxproj

```diff
@@ -418,6 +418,7 @@
 				GCC_PREPROCESSOR_DEFINITIONS = (
 					"DEBUG=1",
 					"$(inherited)",
+					_LIBCPP_ENABLE_CXX17_REMOVED_UNARY_BINARY_FUNCTION,
```

Comment:
i would recommend not to commit this xcode 15 workaround.
soon after 0.73 upgrade, we don't need this anymore.